### PR TITLE
Minor example fixes

### PR DESCRIPTION
**Pull request by QuLogic**
Just fixing a few minor little wrinkles.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
Ready for review/merge from your end?

File: examples/basics/scene/flipped_axis.py [hunk 1]
```diff
@@ -77,7 +78,8 @@ def on_key_press(event):
             cam.up = up
     if event.text == '5':
         cam_toggle = {cam1: cam2, cam2: cam3, cam3: cam4, cam4: cam1}
-        view.camera = cam_toggle.get(view.camera, 'fly')
+        view.camera = cam_toggle.get(view.camera, cam2)
+        print(view.camera.name, 'Camera')
```
Comment (Eric89GXL) [inline examples/basics/scene/flipped_axis.py]: `print(view.camera.name + ' camera')`?

Comment (QuLogic) [inline examples/basics/scene/flipped_axis.py]: Oh, I didn't realize you don't import `print_function` here.

**Comment by QuLogic** [comment]
All set.

**Comment by Eric89GXL** [comment]
Thanks @QuLogic!

**Pull request merged**